README.md

```diff
@@ -39,6 +39,89 @@ bundle install
 rake -T
 ```
 
+## Dockerfile
+
+### Build image
+```
+docker build -t <image_name> .
+```
+### Run tests in Docker
+
+Driver which is used by default is **headless chrome**
+
+Use **docker run** command to create and run container.
+
+- In order to run container by default:
+```
+docker run -d --name <container_name> <image_name>
+```
+
+- To run container using firefox headless browser need to override environment variable:
+**SEXY_SETTINGS**
+```
+docker run -e "SEXY_SETTINGS=driver=headless_firefox" -d --name <container_name> <image_name>
+```  
+
+- Connect to created container using this command:
+```
+docker exec -it <container_name/ID> /bin/bash
+```
+
+- Run tests with the next command using container's terminal:
+```
+bundle exec rake
+```
+
+### Start and Stop containers
+- Use docker commands such as **docker start** and **docker stop** to restart container or stop container:
+```
+docker start <container_name/ID>
+docker stop <container_name/ID>
+```
+
+## Docker compose
+
+### Override default driver
+Add to docker-compose.yml environment variable SEXY_SETTING and define value:
+```
+services:
+  howitzer_example_rspec:
+    build:
+      context: "."
+      dockerfile: Dockerfile
+    environment:
+      SEXY_SETTINGS: 'driver=headless_firefox'  
+```
+In this case you can run test using *firefox headless* instead of default *chrome headless* driver 
+
+### Build and run container with example
+
+- Initialize build, and up the container in detached mode:
+```
+docker-compose -f docker-compose.yml up -d
+```
+- Connect to created container using this command:
+```
+docker compose exec -it howitzer_example_rspec  /bin/bash
+Where 'howitzer_example_rspec' is service's name in yml file
+```
+- Run tests with the next command using container's terminal:
+```
+bundle exec rake
+```
+
+### Stop container
```

Comment:
we do not need this section, keep it simple, no reason to repeat official docker documentation